core/modules/startup/startup.js

```diff
@@ -88,6 +88,7 @@ exports.startup = function() {
 			if($tw.browser) {
 				var pluginTiddler = $tw.wiki.getTiddler(plugins[0]);
 				if(pluginTiddler) {
+					document.documentElement.setAttribute("lang",pluginTiddler.getFieldString("title").replace("$:/languages/",""));
```

Comment:
Would it be possible to avoid using the plugin title, and instead use the "name" field?